src/js/lib.js

```diff
@@ -304,7 +301,7 @@ var isEmpty = function(obj) {
  * @private
  */
 var hasClass = function(element, classToCheck){
-  return ((' ' + element.className + ' ').indexOf(' ' + classToCheck + ' ') !== -1);
+  return ((` ${element.className} `).indexOf(` ${classToCheck} `) !== -1);
```

Comment:
And I like Gary's idea anyway...This is now going to be exported anyway, so a little documentation wouldn't hurt.